annotation.go

```diff
@@ -70,24 +70,24 @@ type AnnotationService interface {
 	// ListAnnotations lists all annotations matching the filter.
 	ListAnnotations(ctx context.Context, orgID platform.ID, filter AnnotationListFilter) (ReadAnnotations, error)
 	// GetAnnotation gets an annotation by id.
-	GetAnnotation(ctx context.Context, orgID, id platform.ID) (*AnnotationEvent, error)
```

Comment:
This and other removals of the `orgID` parameter indicate where it should not be necessary to complete the query.